Makefile

```diff
@@ -9,14 +9,14 @@ all: fast-docker-build
 docker-build:
 	docker build --no-cache -t $(REGISTRY)/$(REPOSITORY):$(DEFAULT_TAG) .
 
-docker-build-tests: docker-build
+docker-build-tests:
 	docker build --no-cache -t supervisor-tests -f Dockerfile.tests .
 
 fast-docker-build:
 	docker build -t $(REGISTRY)/$(REPOSITORY):$(DEFAULT_TAG) .
 
-fast-docker-build-tests: fast-docker-build
-	docker build -t supervisor-tests -f Dockerfile.tests .
+fast-docker-build-tests:
+	./run_tests.sh
```

Comment:
This docker build so you can probably have:
docker build -t supervisor-tests -f Dockerfile.tests .

Reply:
No. The CACHEBUST argument is a necessary part here and I couldn't find any other way with Makefile to perform this.

You cannot assign any value to the variables in a Makefile code on run time. Its the only reason I resorted for this way. I've spent a good time to look for other ways already.